api/mysagw/case/serializers.py

```diff
@@ -115,11 +128,11 @@ def create(self, validated_data):
         return new_accesses
 
     def _send_mail(self, new_accesses_for_user):
-        identity = new_accesses_for_user[0].identity
+        identity = new_accesses_for_user[0][0].identity
         subject = case_transfer.EMAIL_BULK_INVITE_SUBJECTS[identity.language]
         links = "\n".join(
             [
-                f"{settings.SELF_URI}/cases/{access.case_id}"
+                f"{access[1]} - {settings.SELF_URI}/cases/{access[0].case_id}"
                 for access in new_accesses_for_user
```

Comment:
Okay. Finally got what it's about ;)
Maybe a stupid question: Why not fetch the case from the db and read the dossier-number from there?

Reply:
Because it's in a different DB ;)